packages/special-pages/pages/onboarding/app/components/Switch.js

```diff
@@ -16,7 +16,7 @@ import { useEnv } from '../../../../shared/components/EnvironmentProvider'
 export function Switch ({ checked = false, variant, ...props }) {
     const { onChecked, onUnchecked, ariaLabel, pending } = props
     const env = useEnv()
-    const platform = variant || env.platform
+    const platform = variant || env.injectName
```

Comment:
I know this is just a local var, but is it worth changing it to injectName too to further avoid confusion?